frontend/packages/dev-console/integration-tests/features/topology/topology-smoke_test.feature

```diff
@@ -0,0 +1,165 @@
+Feature: Topology smoke test
+	As a user, I want to verify topology feature    
+
+@regression, @smoke
+Scenario: Empty state of topology
+   Given user is in administrative perspective
+   When user goes to developer perspective
+   And user creates a new project
+   Then user sees Add page with message on the top "No workloads found"
+
+@regression, @smoke
+Scenario: Topology with workloads
+   Given user is in administrative perspective
+   When user goes to developer perspective
+   And user selects a existing project from project list with existing workloads
+   Then user sees different workloads in topology chart area
+
+@regression, @smoke
+Scenario: Visual for deployment/deployment-config
+   Given user is in topology with deployment workload
+   When user checks nodes and the decorators associated with them
+   And nodes are circular shaped with builder image in them
```

Comment:
From this to last When statement, I don't see any action performed, can you please change these to Then statements as user is not performing any action and just seeing the  workload

Reply:
But isn't the then statement is used in the last statement